--- acinclude.m4~	2020-11-05 19:15:43.000000000 -0600
+++ acinclude.m4	2021-04-07 17:15:30.000000000 -0500
@@ -362,6 +362,7 @@
   AC_CACHE_CHECK([$1], [$2],
   [AC_TRY_LINK(
   [#include <stdlib.h>
+  #include <string.h>
   #include "bfd.h"
   #include "$4"
   ],
--- configure~	2020-11-05 19:15:43.000000000 -0600
+++ configure	2021-04-07 17:15:44.000000000 -0500
@@ -16818,6 +16818,7 @@
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 #include <stdlib.h>
+  #include <string.h>
   #include "bfd.h"
   #include "elf-bfd.h"
 
